src/tools/compiletest/src/directives.rs

```diff
@@ -959,6 +959,7 @@ const KNOWN_DIRECTIVE_NAMES: &[&str] = &[
     "only-aarch64-unknown-linux-gnu",
     "only-apple",
     "only-arm",
+    "only-arm-unknown-linux-gnueabihf",
```

Comment:
@jieyouxu is this the right thing to do here?

Reply:
To let more targets through (as requested below), we can instead allow `only-eabihf`. That is not perfect, because there are `aebi` (no `hf`) targets that still do support executing floating point instructions, but it at least casts a wider net.

Maybe `ignore-softfloat` could work. I'm not sure how to check for that actually working, from what I can tell the arm targets generally still support floating point instructions, even if they don't pass arguments via fp registers.

Reply:
What I meant is, do we just have to add this here? I am surprised there's no new code implementing the logic for this filter.^^

Reply:
> What I meant is, do we just have to add this here? I am surprised there's no new code implementing the logic for this filter.^^

Yes, that would be sufficient. This filter is only meant to catch unknown directives, but compiletest actually already have logic to accept `//@ only-$target_tuple` for all target tuples.